### DISCUSSION TITLE
How Can we create macos native modules and use it in react-native?

### DISCUSSION THREAD

**Discussion by VarunTwri** [microsoft/react-native-windows#9739] [Q&A] [answered]
How Can we create macos native modules and use it in react-native?

**Comment by chrisglein** [comment, marked as answer]
Sounds like a question for the [react native macOS](https://github.com/microsoft/react-native-macos) repo. FWIW, this is underdocumented on the website though, which is known: [issue here](https://github.com/microsoft/react-native-windows-samples/issues/159).
